home/migrations/0085_orderedcontentset_locale_orderedcontentset_slug.py

```diff
@@ -3,47 +3,53 @@
 from django.db import migrations, models
 from django.db.models import Count
 import django.db.models.deletion
+import home.models
 
 
 def rename_duplicate_slugs(OrderedContentSet):
     duplicate_slugs = (
-        OrderedContentSet.objects.values("slug", "locale")
+        OrderedContentSet.objects.values("slug")
         .annotate(count=Count("slug"))
         .order_by("-count")
         .filter(count__gt=1)
-        .values_list("slug", "locale")
+        .values_list("slug", flat=True)
     )
-    for slug, locale in duplicate_slugs:
-        pages = OrderedContentSet.objects.filter(slug=slug, locale=locale)
-        while pages.count() > 1:
-            page = pages.first()
-
-            suffix = 1
-            candidate_slug = slug
-            while OrderedContentSet.objects.filter(
-                slug=candidate_slug, locale=locale
-            ).exists():
+    for slug in duplicate_slugs:
+        ordered_content_sets = OrderedContentSet.objects.filter(slug=slug)
+        for ordered_content_set in ordered_content_sets:
+            slug_from_name = (
+                ordered_content_set.name.lower().replace(" ", "-").replace("_", "-")
+            )
```

Comment:
Maybe we can use Django's `slugify` function here? https://docs.djangoproject.com/en/5.1/ref/utils/#django.utils.text.slugify . I think slugs are allowed to have underscores

Reply:
Didn't know that existed. Thanks!